src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,36 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion (val ciudad: Ciudad){
+
+    val vendedoresDelCentro = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(Vendedor : Vendedor) {
+
+        if (vendedoresDelCentro.contains(Vendedor)) {
+            throw Exception("este vendedor ya se encuentra en el centro")
+        }else{
+            vendedoresDelCentro.add(Vendedor)
+        }
+    }
```

Comment:
Si entra por la condición del `if` se va a cortar la ejecución por lo que no es necesario poner un `else`
```suggestion
    fun agregarVendedor(Vendedor : Vendedor) {
        if (vendedoresDelCentro.contains(Vendedor)) {
            throw Exception("este vendedor ya se encuentra en el centro")
        }
        vendedoresDelCentro.add(Vendedor)
    }
```